src/basis/Monad.ml

```diff
@@ -12,6 +12,7 @@ module type Notation = sig
   val (let*) : 'a m -> ('a -> 'b m) -> 'b m
   val (let+) : 'a m -> ('a -> 'b) -> 'b m
   val (and+) : 'a m -> 'b m -> ('a * 'b) m
+  val (and*) : 'a m -> 'b m -> ('a * 'b) m
```

Comment:
lol I didn't know this exists!

Reply:
https://caml.inria.fr/pub/docs/manual-ocaml/bindingops.html#ss:letops-rationale